src/templates/pages/editprefset-topnavbar-widget.handlebars

```diff
@@ -0,0 +1,43 @@
+<div class="top-bar">
+    <div class="top-bar-left">
+        <ul class="menu">
+            <li>
+                <a href="/ppt"><i class="pmt-topnav-icon fi-home"></i></a>
+            </li>
+            <li>
+                <h2 id="top">Preference Set: {{npsetName}}</h2>
+            </li>
+        </ul>
+    </div>
+    <div class="top-bar-right">
+        <ul class="menu">
+            <li>
+                {{#if devModeOn}}
+                    <a class="pmt-topnav-icon-active" id="pmt-topbar-devmode-button"><i class="fi-wrench"></i></a>
+                {{else}}
+                    <a class="pmt-topnav-icon" id="pmt-topbar-devmode-button"><i class="fi-wrench"></i></a>
+                {{/if}}
+            </li>
+            <li>
+                <a class="pmt-topnav-icon" id="pmt-topbar-download-button"><i class="fi-download"></i></a>
+            </li>
+            <li>
+                {{#if unsavedChangesExist }}
+                    <a class="pmt-topnav-icon-active" id="pmt-topbar-save-button"><i class="fi-save large"></i></a>
+                {{else}}
+                    <a class="pmt-topnav-icon" id="pmt-topbar-save-button"><i class="fi-save large"></i></a>
+                {{/if}}
+            </li>
+            <!-- <li> -->
```

Comment:
Remove the commented code if it's no longer in use. Thanks.

Reply:
removed